gazebo_ros2_control_demos/package.xml

```diff
@@ -33,7 +33,6 @@
   <exec_depend>robot_state_publisher</exec_depend>
```

Comment:
> I can see that the package effort_controllers was released but it seems that it not still synchronized.

Nice, I installed it from `ros2-testing` and now the demo works for me. Should we add it as an `exec_dependency` here?

Reply:
Added an issue https://github.com/ros-simulation/gazebo_ros2_control/issues/49